mon-pix/tests/integration/components/campaign/invited/learner-reconciliation_test.js

```diff
@@ -0,0 +1,129 @@
+import { render } from '@1024pix/ember-testing-library';
+import { click, fillIn } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+import { module, test } from 'qunit';
+import sinon from 'sinon';
+
+import setupIntlRenderingTest from '../../../../helpers/setup-intl-rendering';
+
+module('Integration | Component | Campaign | Invited | learner-reconciliation', function (hooks) {
+  setupIntlRenderingTest(hooks);
+  let reconciliationFields, organizationName, onSubmit;
+
+  hooks.beforeEach(function () {
+    onSubmit = sinon.stub();
+    organizationName = 'My Organization';
+    reconciliationFields = [
+      {
+        key: 'field2',
+        columnName: 'Prénom',
+      },
+      {
+        key: 'field1',
+        columnName: 'Date de naissance',
+      },
+    ];
+
+    this.set('reconciliationFields', reconciliationFields);
+    this.set('organizationName', organizationName);
+    this.set('onSubmit', onSubmit);
+  });
+
+  test('it should display reconciliation form', async function (assert) {
+    // given / when
+    const screen = await render(
+      hbs`<Campaigns::Invited::LearnerReconciliation
+        @reconciliationFields={{this.reconciliationFields}}
+        @organizationName={{this.organizationName}}
+      />`,
+    );
+    // then
+    assert.ok(
+      screen.getByRole('heading', {
+        name: this.intl.t('components.invited.reconciliation.title', { organizationName }),
+        level: 1,
+      }),
+    );
+    assert.ok(screen.getByText(this.intl.t('common.form.mandatory-all-fields')));
+
+    assert.ok(screen.getByRole('textbox', { name: 'Prénom' }));
+    assert.ok(screen.getByRole('textbox', { name: 'Date de naissance' }));
+    assert.ok(screen.getByRole('button', { name: this.intl.t('common.actions.lets-go') }));
+  });
+
+  module('validation cases', function () {
```

Comment:
**To do :** ajouter un test pour le cas où l'utilisateur ne peux pas se réconcilier (erreur date de naissance ou nom,...)